.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,24 @@
+name: Feature Request
+description: File a feature request
+title: "[Feature]: "
+labels: ["enchancement"]
+body:
+  - type: textarea
+    id: detail
+    attributes:
+      label: Detail
+      description: Please explain the feature in details.
+    validations:
+      required: true
+  - type: textarea
+    id: reason
+    attributes:
+      label: Reason
+      description: Reason for adding this feature.
```

Comment:
Please give an explanation for why you want this feature in the API, as opposed to something you would write yourself in your own mod.